.circleci/config.yml

```diff
@@ -117,8 +117,11 @@ workflows:
             - build
           matrix:
             parameters:
-              jdk: ["8", "11"]
-              kotlin: ["1.5.0", "1.4.32"]
+              jdk: ["8", "11", "17"]
+              kotlin: ["1.4.32", "1.5.31", "1.6.0-RC"]
+            exclude:
+              - jdk: "17"
```

Comment:
don't we need Gradle-7.3-rc to even run on 17?

Reply:
Sure looks like it ran using 17: https://app.circleci.com/pipelines/github/open-toast/protokt/657/workflows/e215faaf-7e39-4a46-84f9-348eca4e754a/jobs/1084

Reply:
well anyway 7.3 is out